src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/BaseRecalibratorWorker.java

```diff
@@ -215,6 +215,14 @@ public StandardCovariateList getRequestedCovariates() {
         return this.covariates;
     }
 
```

Comment:
Can't comment on lines that aren't in this pull request, but the docs in this file still refer to `BaseRecalibratorUprooted`